File: others/stack.c
#include<stdio.h>

int i = -1;
int arr[10];

void push()
{
    if(i == 10)
    {
        printf("Stack Overflow\n");
    }
    else
    {
        i++;
        printf("Enter a number: ");
        scanf("%d",&arr[i]);
    }
}

void pop()
{
    if(i == -1)
    {
        printf("Stack underflow\n");
    }
    else
    {
        printf("Stack removed: %d\n",arr[i]);
        i--;
    }
}

void display()
{
    int j;
    printf("Stack elements:");
    for(j = 0; j <= i; j++)
    {
        printf(" %d",arr[j]);
    }
    printf("\n");
}

int main()
{
    int n = 1;
    while(n)
    {
        printf("\t Operation for stack: - \n");
        printf("\t 	1. PUSH \n");
        printf("\t 	2. POP \n");
        printf("\t 	3. DISPLAY \n");
        printf("\t 	4. EXIT \n");
        printf("Enter your option: ");
        int a;
        scanf("%d",&a);
        switch(a)
        {
            case 1:
                    push();
                    break;
            case 2:
                    pop();
                    break;
            case 3:
                    display();
                    break;
            case 4:
                    printf("Operation ended\n");
                    n--;
                    break;
            default:
                    printf("You entered wrong option.\n");
        }
    }
}

File: others/STACK.py
stack_limit = 10	# stack size limit
stack = []

def push():
    if stack_limit == len(stack):
        print("STACK Overflow\n")
    else:
        stack.append(input("PUSH ELEMENT: "))

def pop():
    if len(stack) == 0:
        print("STACK Underflow\n")
    else:
        print("POP ELEMENT:",stack.pop())

def display():
	print("STACK:",stack)

if __name__ == '__main__':
    while(True):
        print("\t Operation for STACK: -")
        print("\t 	1. PUSH")
        print("\t 	2. POP")
        print("\t 	3. DISPLAY")
        print("\t 	4. EXIT")
        option = int(input("Enter your option: "))
        if option == 1:
            push()
        elif option == 2:
            pop()
        elif option == 3:
            display()
        elif option == 4:
            break
